src/assets/sass/components/ae-toolbar/skin.scss

```diff
@@ -23,7 +23,7 @@
 
 .ae-toolbar-add {
     background-color: $toolbar-add-bg-color;
-    border-color: $toolbar-add-border-color;
+    outline: solid $toolbar-add-border-width $toolbar-add-border-color;
```

Comment:
Looks like CSS is separated here in AE into style and structure files. Interesting, not sure why. Looks like a lot a hassle for little gain. Maybe the idea is to have style-only skins? In any case, it's probably best to follow the existing pattern, meaning outline width should be in structure. It should also be sorted.

Reply:
@orsolyaDekany  It was ok to modify the skin. The pattern of code in AE is to separate styles:
- those that affect layout, like `width`, are in `structure.scss`
- those that do no affect layout, like `color`, are in `skin.scss`

Both are styles on the same `.ae-toolbar-add`.

I think it's best we follow the same pattern of code placement. This does not mean placing all styles in `skin.scss`, as in previous solution, or placing all styles in `structure.scss`, as in current solution.

Reply:
Hi @markocikos , I see. The structure is a bit confusing. I updated the PR, the color is remained in the skis.scss and other styles are in the sutrcture.scss, I think it is following the pattern. Let me know. Thanks.